sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -144,7 +165,10 @@ def _sale_done(self, sale_done_filter):
     def run_with_workflow(self, sale_workflow):
         workflow_domain = [("workflow_process_id", "=", sale_workflow.id)]
         if sale_workflow.validate_order:
-            self._validate_sale_orders(
+            _self = self.with_context(
+                send_order_confirmation_mail=sale_workflow.send_order_confirmation_mail
+            )
+            _self._validate_sale_orders(
```

Comment:
Why not doing it in a single statement? Isn't it a little bit confusing this way?

Reply:
Although it would be a good idea, I think it is not possible because there is no field similar to `send_confirmation_mail_date` to help us not to send email several times.
Can you think of a way to do it?